src/Components/Molecules/FMSelect/index.tsx

```diff
@@ -31,7 +26,7 @@ export const FMSelect = memo((props: ISelectInput) => {
     adornment && adornmentPosition === "start" ? AddAdornment() : null;
 
   return (
-    <FormControl sx={{ minWidth: 130 }} variant="outlined" data-testid={testID}>
+    <>
       <InputLabel id="demo-customized-select-label">{label}</InputLabel>
```

Comment:
better to use InputLabel from atoms component and use dynamic id

Reply:
i decide not to use it since it working just fine with the new added FATextField from the Atoms